examples/file_to_pipe.exs

```diff
@@ -0,0 +1,19 @@
+import Membrane.ChildrenSpec
+import Membrane.Testing.Assertions
+alias Membrane.Testing.Pipeline
```

Comment:
This is intended to be used in tests, let's use a regular pipeline or `Membrane.RCPipeline` instead